site/content/concepts/device-notifications.md

```diff
@@ -17,86 +17,65 @@ Please join our mailing list or consult the IRC channel in case you have any que
 <!--more-->
 
 For implementing specific behaviour *Business applications* need to know if
+a device is currently ready to receive an upstream message.
 
-- a device is currently ready to receive a command
-- a device is currently connected to a protocol adapter
- 
-The first can serve e.g. to implement functionality like sending a command upstream to a device.
-
-In the following it is described how a *Business application* can be notified that a device is ready to receive a command. 
-This is referred to as `device command readiness`.
+This is the purpose of the following *Time until disconnect notification*.
 
-## Device command readiness notification
+## Time until disconnect notification
 
-To notify an application about the command readiness of a device specific properties are set in a downstream message.
+This notification can be triggered in an application by any downstream message as long as the following AMQP 1.0 properties are set:
 
-This can be done
+- the property `creation-time` (automatically set by protocol adapters to the current time when creating the message)
+- an additional application-property `ttd` (the number of seconds the device will stay connected to the protocol adapter)
 
-- implicitly: when sending a downstream message for any reason (e.g. telemetry or event message)
-- explicitly: by sending a specific empty event (e.g. if a device does not have any other downstream message to send)
+A device that wants to only trigger this notification (without sending other downstream data) may send an 
+*empty notification* event.
```

Comment:
what about a link to the specification of the *empty notification* in the Event API?